jenkins/job-configs/kubernetes-jenkins-pull/bootstrap-maintenance-pull.yaml

```diff
@@ -44,7 +44,11 @@
         repo-name: 'k8s.io/test-infra'
     - janitor:
         branch: master
-        frequency: '0 */3 * * *'
+        frequency: 'H */3 * * *'
         job-name: maintenance-pull-janitor
         repo-name: 'k8s.io/test-infra'
-
+    - aws-janitor:
+        branch: master
+        frequency: 'H * * * *'
+        job-name: maintenance-aws-janitor
```

Comment:
How do we fix that? The only way to tell if resources are getting leaked is if this job fails.